tests/phpunit/tests/rest-api/rest-navigation-fallback-controller.php

```diff
@@ -143,6 +143,49 @@ public function test_adds_links() {
 		$this->assertTrue( $links['self'][0]['attributes']['embeddable'], 'Self link should be embeddable.' );
 	}
 
+	/**
+	 * Tests that the correct filters are applied to the context parameter.
+	 *
+	 * By default, the REST response for the Posts Controller will not return all fields
+	 * when the context is set to 'embed'. Assert that correct additional fields are added
+	 * to the embedded Navigation Post, when the navigation fallback endpoint
+	 * is called with the `_embed` param.
+	 *
+	 * @ticket 58557
+	 * @covers wp_add_fields_to_navigation_fallback_embedded_links
+	 *
+	 * @since 6.3.0 Added Navigation Fallbacks endpoint.
+	 */
+	public function test_embedded_navigation_post_contains_required_fields() {
+		// First we'll use the navigation fallback to get a link to the navigation endpoint.
+		$request  = new WP_REST_Request( 'GET', '/wp-block-editor/v1/navigation-fallback' );
+		$response = rest_get_server()->dispatch( $request );
+		$links    = $response->get_links();
+
+		// Extract the navigation endpoint URL from the response.
+		$embedded_navigation_href = $links['self'][0]['href'];
+		preg_match( '/\?rest_route=(.*)/', $embedded_navigation_href, $matches );
+		$navigation_endpoint = $matches[1];
+
+		// Fetch the "linked" navigation post from the endpoint, with the context parameter set to 'embed' to simulate fetching embedded links.
+		$request = new WP_REST_Request( 'GET', $navigation_endpoint );
+		$request->set_param( 'context', 'embed' );
```

Comment:
I've ported this test as-is. 

It failed locally unless I changed this line to:

```
$request->set_param( 'context', 'edit' );
```

Any ideas?

Reply:
```
1) WP_REST_Navigation_Fallback_Controller_Test::test_embedded_navigation_post_contains_required_fields
Response title should contain a "status" field.
Failed asserting that an array has the key 'status'.
```

Reply:
If it doesn't work it probably means that the filter on `rest_wp_navigation_item_schema` isn't being called. Is the file being included?

The whole point of the filter is that if you make a request for a single Navigation Menu and change the context to `embed` you will get these additional fields.

That's what the test does.

Reply:
Thanks @getdave 

You know what: it wasn't imported 🤣 

The file had already been backported, so I assumed it was.

But it wasn't.

So here we are. 

Thanks for the tip!